x-pack/test/accessibility/apps/rollup_jobs.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'settings', 'header', 'rollup']);
+  const a11y = getService('a11y');
+  const testSubjects = getService('testSubjects');
+  const kibanaServer = getService('kibanaServer');
+  const esArchiver = getService('esArchiver');
+  const es = getService('es');
+
+  describe('Stack management- rollup a11y tests', () => {
+    const rollupJobName = `rollup${Date.now().toString()}`;
+    before(async () => {
+      await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/logstash_functional');
+      await kibanaServer.uiSettings.update({
+        defaultIndex: 'logstash-*',
+      });
+      await PageObjects.settings.navigateTo();
+      await PageObjects.rollup.clickRollupJobsTab();
+    });
+
+    it('empty state', async () => {
+      await a11y.testAppSnapshot();
+    });
+
+    describe('create a rollup job wizard', async () => {
+      it('step 1', async () => {
```

Comment:
Just a question man, when I see phrases like `step #`, it makes me think there's a lot more context going on.
Now this is just a nit, but is it useful and / or possible to name these steps more comprehensively?
If not, no worries my dude.